aten/src/ATen/native/NNPACK.cpp

```diff
@@ -0,0 +1,574 @@
+#include <ATen/Config.h>
+#include <ATen/ATen.h>
+
+#if !AT_NNPACK_ENABLED()
+
+namespace at { namespace native {
+
+at::Tensor nnpack_spatial_convolution(
```

Comment:
Eh we should figure out a better place for these files than a `native` directory. There's really nothing native in this code...

Reply:
My understanding is that 'native' simply refers to any function whose binding code is implemented in ATen, as opposed to the legacy TH. This is why @zdevito has been so adamant about not exposing native to the public interface (we now have the abstract/concrete distinction for that). Does "native" have a meaning beyond "concrete"? If not, then perhaps what you are asking for is a rename. (I do think 'native' gives the wrong intuition).